ts/component/table-builder.ts

```diff
@@ -306,6 +315,13 @@ export class TableBuilder<ParentType, ChildType> {
     return this;
   }
 
+  private addSelectionCount(container: HTMLElement, countElement: HTMLElement) {
+    countElement.className =
+      "selection-count font-inter font-medium text-12 text-black bg-grey-100 px-2 py-1 rounded-md cursor-default inline-block";
```

Comment:
The grey background is used for clickable elements (buttons, dropdowns). This text should be styled similar to the item counts in the paging controls